x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -72,3 +73,6 @@ export function assert(condition: any, msg?: string): asserts condition {
     throw new Error(msg);
   }
 }
+
+export const calculatePostureScore = (passed: number, failed: number): Score =>
+  roundScore(passed / (passed + failed));
```

Comment:
```suggestion
export const calculatePostureScore = (passed: number, failed: number): Score => { 
const total = passed + failed
if (total === 0) return 0 
return roundScore(passed/total)
}
```

this is to avoid passing `Infinity` when `passed+failed` is `0`